### DISCUSSION TITLE
Get all records via REST API

### DISCUSSION THREAD

**Discussion by bliepp** [pocketbase/pocketbase#5097] [Q&A] [answered]
Hey there,

so I'm currently trying to access a pocketbase instance via the REST API (I'm using python, so none of the SDKs are an option). When I call `/api/collections/<collection>/records` there are only 30 records in the response (so far, so documented). If I set the `perPage`to something ridiculously high (e.g. 10k) I still get only 500 records. How do I get all records in my collection? I know the SDKs have a `getFullList`method but I'm still struggling on how transfer this to my python code.

Best,
bliepp

**Comment by ganigeorgiev** [comment, marked as answer]
See https://github.com/pocketbase/pocketbase/discussions/4660.

**Reply by bliepp** [reply]
Oh, I searched the discussions but didn't find that post. Thanks.
